README.md

```diff
@@ -28,28 +36,22 @@ You should get the orderFormId and then you may inject that in _vtexcommercestab
 
 ## Responding to checkout
 
-The checkout API expects a response from the app through the `transactionValidation.vtex` event, therefore an approach for responding is:
+The checkout UI API expects a response from the app through the `transactionValidation.vtex` event, therefore an approach for responding is:
 
-```js
+```
 $(window).trigger('transactionValidation.vtex', [status])
 ```
 
-Where `status` is a boolean and resolves(`status == true`) or rejects (`status == false`) order placement.
+Where `status` is a boolean and resolves(`status == true`) or rejects (`status == false`) order placement. At this point, the checkout UI will validate if the transaction was `approved` or `denied` and according to this status if the transaction is approved checkout UI will redirect to the Order Placed page, otherwise will show a warning modal.
```

Comment:
...transaction was `approved` or `denied` **(note: it will check on transactions API)**...

Reply:
It's not intended to show how checkout ui works under the hood.